spec/views/questions/show.html.haml_spec.rb

```diff
@@ -5,7 +5,7 @@
     before do
       assign(:question,
         double(:question,
-          asked_at_date: Date.parse('24 Nov 2017'),
+          asked_at_date: Time.parse('24 Nov 2017'),
```

Comment:
Is the conversation from Date to Time important for the `l` function?

Reply:
The unit test was breaking when I set the time value to use `:default`. With the Date class, it was looking for `en.date.formats.default` in the translation. I could have fixed that by adding a default value into the translation but we already have a `en.time.formats.default`, which provides the default value. That gets targeted with the Time class. This is how the translation currently looks:

```
time:
    formats:
      default: '%-d %B %Y'
  date:
    abbr_month_names: ['', 'Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun', 'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec']
```

I would have to do this to make the test pass with the Date class:

```
time:
    formats:
      default: '%-d %B %Y'
  date:
    abbr_month_names: ['', 'Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun', 'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec']
    formats:
      default: '%-d %B %Y'
```

Reply:
That's fine, leave the translations as they are - based on this I would assume that when GROM deserialises dates from ntiple files, they are created as Time objects rather than Date. In which case adding an additional date format block would be an incorrect thing to do